lib/std/fifo.zig

```diff
@@ -153,7 +153,7 @@ pub fn LinearFifo(
             var start = self.head + offset;
             if (start >= self.buf.len) {
                 start -= self.buf.len;
-                return self.buf[start .. self.count - offset];
+                return self.buf[start .. start + (self.count - offset)];
```

Comment:
Might be better written as `self.buf[start..][0..self.count - offset]`

Reply:
Eh, I think I like the original approach better.